plugins/outputs/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -162,7 +161,10 @@ func (c *CloudWatchLogs) getDest(t Target, logSrc logs.LogSrc) *cwDest {
 			c.Log.Info("Configured middleware on AWS client")
 		}
 	}
-	pusher := NewPusher(c.Region, t, client, c.ForceFlushInterval.Duration, maxRetryTimeout, c.Log, c.pusherStopChan, &c.pusherWaitGroup, logSrc)
+
+	configurer := awsmiddleware.NewConfigurer(c.middleware.Handlers())
+	pusher := NewPusher(c.Region, t, client, c.ForceFlushInterval.Duration, maxRetryTimeout, c.Log, c.pusherStopChan, &c.pusherWaitGroup, logSrc, configurer)
```

Comment:
Passing in Configurer which has the agent health handlers

Reply:
Why do we need to pass in the configurer? The `cloudwatchlogs` output plugin already configures the CWL client.